test/plot-schema.json

```diff
@@ -16463,6 +16475,18 @@
       "inside bottom"
      ]
     },
+    "ticklabelrunoff": {
+     "description": "Shifts the tick labels by the specified number of pixels in parallel to the axis.",
+     "dflt": 0,
+     "editType": "ticks",
+     "valType": "integer"
+    },
+    "ticklabelstandoff": {
+     "description": "Shifts the tick labels by the specified number of pixels orthogonally to the axis.",
```

Comment:
Let's mention in the description that the positive values are moving the ticks towards the positive direction of the axis, etc.

Reply:
Clarification was added.

Reply:
Thanks. Resolving.